### PR TITLE
feat: homepage added

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
- Sidebar added
- Navbar Added
- Notes form added
- Notes card added
## Screenshots
Screenshot (230)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,9 +1,15 @@
 :root {
   --primary-color: #2232cf;
   --text-color: black;
+  --white-color: #fff;
+  --icons-color: #5f5e5e;
```
Comment (sharmayatendra) [inline src/index.css:5]: try to use only one convention for colors

File: src/pages/HomePage/HomePage.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  margin: 0 5rem;
+}
+
+.homepage-main-section {
+  flex-grow: 1;
+  margin: 1.5rem 0;
+}
+
+.homepage-search-input-wrapper {
+  border: 3px solid #d1d1d1;
```
Comment (sharmayatendra) [inline src/pages/HomePage/HomePage.css:11]: you can define this as a variable

**Review by sharmayatendra — COMMENTED** [review]
The code is clean, has a good file structure, you have followed all the best practices. Gook work!

**Review by yashpatel1797 — APPROVED** [review]
- File structure is good.
- code looks clean.
- design is good.

File: src/index.css (hunk 1 above)
Comment (devdeadviz) [reply src/index.css:5]: Okay

File: src/pages/HomePage/HomePage.css (hunk 2 above)
Comment (devdeadviz) [reply src/pages/HomePage/HomePage.css:11]: I have used this color only here, so that's why I haven't created a variable for that.

**Pull request merged**